codecov.yml

```diff
@@ -0,0 +1,41 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  status:
+    project:
+      default:
+        target: 90% #overall project/ repo coverage
+      frontend:
+        target: 90%
+        flags:
+          - frontend
+      backend:
+        target: 95%
+        flags:
+          - backend
```

Comment:
Added this `codecov` config with our respective coverage targets for the frontend and backend.